theme_v2_json_structure.py

```diff
@@ -0,0 +1,366 @@
+                            },
+                        }
+                    ]
+                },
+            ]
+        },
+        "courses": {
+            #--#--  API key: pages_courses
+        },
+        "about_us": {
+            #--#--  API key: pages_about_us
+        },
+        "terms": {
+            #--#--  API key: pages_terms
+        },
+        "contact_us": {
+            #--#--  API key: pages_contact_us
+        },
+        "pages_custom_page_1": {
+            #--#--  API key: pages_custom_page_1
+            "sections": [
+                # .... the rest of the properties
+            ],
+            #--#-- other details similar to home
+        },
+        "pages_custom_page_2": {
+            #--#--  API key: pages_custom_page_2
+            "slug": "unique-custom-slug",
+
+            "sections": [
+                # .... the rest of the properties
+            ],
+            #--#-- other details similar to home
+        },
+        "pages_custom_page_3": {  #--  API key: pages_custom_page_3
+            "slug": "unique-custom-slug",
+
+            "sections": [
+                # .... the rest of the properties
+            ],
+            #--#-- other details similar to home
+        },
+
+        "pages_custom_page_4": {},  #--  API key: pages_custom_page_4
+
+        "pages_custom_page_5": {},  #--  API key: pages_custom_page_5
+
+        "pages_custom_page_6": {},  # --  API key: pages_custom_page_6
+
+        "pages_custom_page_7": {},  #--  API key: pages_custom_page_7
+
+        "pages_custom_page_8": {},  #--  API key: pages_custom_page_8
+    },
+
+    "custom_pages": [  #--#--  API key: custom_pages, maximum 8 pages
+        "pages_custom_page_1",
+        "pages_custom_page_2",
+        "pages_custom_page_5"
+    ],
+
+    "visual_identity": {
+        "colors": {
+            "primary_color: "#00ff00",  #--  API key: colors_primary_color
+            "secondary_color: "#00ff00",  #--  API key: colors_secondary_color
+        },
+        "fonts": {
+            "heading": {  #--  API key: fonts_heading
+                "ar":"",
+                "en":""
+            },
+            "text": {  #--  API key: fonts_text
+                "ar":"",
+                "en":""
+            }
+        }
+    },
+
+    "platform_settings": {  #--  API key: platform_settings
+        "language": {  #--  API key: platform_settings_language
+            "languages": [
+                "ar", "en", "fr"
+            ],
+            "default_language": "ar",
+        },
+        "site_name": {
+            "ar": "المعهد العقاري",
+            "en": "SREI excellence"
+        },
+        "subtitle": {
+            "ar": "المعهد العقاري",
+            "en": "SREI excellence"
+        },
+        "favicon": "url",
+        "thumbnail": "url",
+        "social_media_image": "url",
+        "meta_description": {
+            "ar": "المعهد العقاري",
+            "en": "SREI excellence"
+        },
+    },
+
+    "footer": {
```

Comment:
add dark/light style mode
```suggestion
    "footer": {
       "style": "dark",
```

Reply:
will the style be applied to the footer only?

Reply:
> will the style be applied to the footer only?

@shadinaif, I'm thinking of `"template": "dark-v1"` and `"template": "light-v1"`. so we add more templates in the future as requested.